### DISCUSSION TITLE
n.getFiberRoots is not a function bug ??

### DISCUSSION THREAD

**Discussion by mtahagocer** [vercel/next.js#18121] [Help]
Hi to every one. How to fix this. I did nothing for this and error show up randomly.
Unhandled Runtime Error
TypeError: n.getFiberRoots is not a function

my package.json
"dependencies": {
    "@material-ui/core": "^4.11.0",
    "@material-ui/icons": "^4.9.1",
    "@material-ui/lab": "^4.0.0-alpha.56",
    "@zeit/next-css": "^1.0.1",
    "axios": "^0.20.0",
    "bcryptjs": "^2.4.3",
    "bootstrap": "^4.5.3",
    "firebase": "^7.24.0",
    "fs": "0.0.1-security",
    "jsonwebtoken": "^8.5.1",
    "next": "^9.5.5",
    "react": "^16.14.0",
    "react-dom": "^16.14.0",
    "react-image-gallery": "^1.0.8",
    "reactstrap": "^8.6.0",
    "slugify": "^1.4.5"
  }

**Comment by ljosberinn** [comment]
Could you please provide the entire stack trace of the error and/or a minimum reproduction repository?
